index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <title>Néstor Oro</title>
+        <link rel="stylesheet" type="text/css" href="assets/css/style.css"/>
+    </head>
+    <body>
+        <header>
+            <div class="toolbar">
+                <a href="" class="logo-placeholder">
+                    <h3>Welcome</h3>
+                </a>
+                <button type="button" class="menu">
+                    <img src="assets/img/menu.svg" alt="menu link">
+                </button>
+            </div>
```

Comment:
- [ ] Kindly close the `header` tag after the hamburger button and please 🙏🏽 put the hamburger button inside a `nav` tag for the best practices of HTML&CSS.